app/code/core/Mage/Eav/Model/Config.php

```diff
@@ -484,6 +484,11 @@ public function getAttribute($entityType, $code, $storeId = null)
             $attribute = $this->_getDefaultAttributeIfExists($entityType, $code, $storeId);
         }
 
+        // maintain old behavior and return an empty model to avoid breaking compatibility
```

Comment:
```suggestion
```

Reply:
I think it should stay, as there is no clear explanation why an empty model is returned instead of false otherwise.

Reply:
restored it but slightly changed